libMobileAgent/src/Hex/include/Hex/generated/agent-data_generated.h

```diff
@@ -27,15 +27,17 @@ enum Platform {
   Platform_Android = 0,
   Platform_iOS = 1,
   Platform_tvOS = 2,
+  Platform_watchOS = 3,
   Platform_MIN = Platform_Android,
-  Platform_MAX = Platform_tvOS
+  Platform_MAX = Platform_watchOS
```

Comment:
This is a generated file. We need to change the file that generates it

Reply:
🤷 I was able to report a hex with watchOS with this change.

Reply:
The flatbuffer schema file is contained in https://source.datanerd.us/mobile/mobile_flatbuffer_schemas which has had watchOS platform added.

Reply:
Please give me a helpful suggestion as to where the file that generates it is then. The change I made works well.